recipes/geotexxx/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "geotexxx" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/geotexxx-{{ version }}.tar.gz
+  sha256: 0ca0c47e8760e8a2d0ebeb08956aa4674daa3a0ee2870e361c7ecbba5fa70538
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python
+    - requests
+    - importlib-metadata  # [py<38]
```

For noarch--better to just depend on importlib-metadata unconditionally:
```suggestion
    - python >=3.7
    - setuptools
    - pip
  run:
    - python >=3.7
    - requests
    - importlib-metadata
```